parquet-fuzz/.gitignore

```diff
@@ -0,0 +1,3 @@
+Cargo.lock
```

Comment:
Can we ignore the `out` directory? are they just temporary output files?

Reply:
done